logviewer/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'com.google.cloud.tools.jib' version '3.1.4'
+    id "org.springframework.boot" version "2.5.4"
+    id "io.spring.dependency-management" version "1.0.11.RELEASE"
+    id "com.vaadin" version "21.0.1"
+    id 'java'
+}
+
+defaultTasks("clean", "bootRun")
+
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://maven.vaadin.com/vaadin-addons")
+    }
+}
+
+configurations {
+    developmentOnly
+    runtimeClasspath.extendsFrom(developmentOnly)
+}
+
+dependencies {
+    implementation 'com.vaadin:vaadin-core:21.0.1'
+    implementation 'com.vaadin:vaadin-spring:18.0.0'
+    // https://mvnrepository.com/artifact/org.vaadin.artur/a-vaadin-helper
+    implementation 'org.vaadin.artur:a-vaadin-helper:1.7.1'
+    implementation 'org.vaadin.crudui:crudui:4.6.0'
+    implementation 'com.vaadin.componentfactory:enhanced-dialog:21.0.0'
+
+
+    implementation 'org.springframework.boot:spring-boot-starter-web:2.5.4'
+    implementation 'org.springframework.boot:spring-boot-starter-security:2.5.4'
+    implementation 'org.springframework.boot:spring-boot-starter-oauth2-client:2.5.4'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.5.4'
+
+    // https://mvnrepository.com/artifact/com.h2database/h2
+    runtimeOnly 'com.h2database:h2:1.4.200'
+
+    developmentOnly 'org.springframework.boot:spring-boot-starter-actuator:2.5.4'
+    developmentOnly 'org.springframework.boot:spring-boot-devtools:2.5.4'
+}
+
+tasks.jib.dependsOn(tasks.vaadinBuildFrontend)
+jib {
+    from.image = 'eclipse-temurin:17'
```

Comment:
Not 100% sure but did we define the version somewhere else in a more global context already? In that case, please re-use existing variables and dont write it again.

Reply:
It has not been globally defined yet